content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -0,0 +1,66 @@
+---
+reviewers:
+- maplain
+title: Service Internal Traffic Policy
+content_type: concept
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Service Internal Traffic Policy_ enables internal traffic restrictions to only route
+internal traffic to endpoints within the node the traffic originated from. The
+"internal" traffic here refers to traffic originated from Pods in the current
+cluster. This can help to reduce costs and improve performance.
+
+<!-- body -->
+
+## Using Service Internal Traffic Policy
+
+You can enable Internal Traffic Policy for a Service by setting the
+`spec.internalTrafficPolicy` to "Local". This tells kube-proxy to only use node
+local endpoints for cluster internal traffic. Importantly, this Service will
+appear to have zero endpoint for any Pod accessing this Service on a node, when
+this Service doesn't have any local endpoint on the node, but does have endpoints
```

Comment:
This sentence is a little awkward.  How about:

Importantly, for pods on nodes with no endpoints for a given Service, the Service will behave as if it has zero endpoints (for Pods on this node) even if the Services does have endpoints on other nodes.

??

Reply:
updated.